pkg/command/domain/human_readable_flags.go

```diff
@@ -0,0 +1,64 @@
+// Copyright © 2019 The Knative Authors
```

Comment:
@ZhuangYuZY I think the license should be 2020.

Reply:
agree, code change.

Reply:
Hi @ZhuangYuZY , looks like this one is missed?

Reply:
Not sure what happens. I already made the change of 2019 --> 2020. You can check "Files changed" tab.